lib/matplotlib/axes/_axes.py

```diff
@@ -3076,8 +3076,7 @@ def get_next_color():
             if shadow:
                 # Make sure to add a shadow after the call to add_patch so the
                 # figure and transform props will be set.
-                shad = mpatches.Shadow(w, -0.02, -0.02)
-                shad.set(zorder=0.9 * w.get_zorder(), label='_nolegend_')
+                shad = mpatches.Shadow(w, -0.02, -0.02, label='_nolegend_')
```

Comment:
Don‘t we need *zorder*?

Reply:
`Shadow` sets a default zorder (immediately below the parent patch) that's better than this manually calculated one.